build.gradle

```diff
@@ -244,6 +244,7 @@ project(':core') {
   dependencies {
     compile project(':clients')
     compile project(':log4j-appender')
+    compile project(':tools')
```

Comment:
Oh, it's to add it to `kafka//libs` folder from your description. It seems a bit strange for `core` to depend on `tools` though.